frontend/src/hooks/useContractQueries.ts

```diff
@@ -100,7 +100,7 @@ export function useContractQueries() {
         const tx: TransactionItem = {
           contractAddress: result?.data.contract_address,
           localContractId: contract.value?.id ?? '',
-          txId: result?.data.transaction_id,
+          hash: result?.data.transaction_hash,
           type: 'deploy',
           status: 'PENDING',
           data: {},
```

Comment:
nit: result is not null in this context
```suggestion
          contractAddress: result.data.contract_address,
          localContractId: contract.value?.id ?? '',
          hash: result.data.transaction_hash,
          type: 'deploy',
          status: 'PENDING',
          data: {},
```